### PR TITLE
Add access to direction argument of scales::brewer_pal

### PR DESCRIPTION
**Pull request by jiho**
Plus minor example and documentation fixes

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,7 @@
+* Add access to the new `direction` argument of `scales::brewer_pal` and
```
Comment (hadley) [inline NEWS]: Sorry I think I was unclear bfore, this should be the first bullet under the version heading

Comment (jiho) [inline NEWS]: Oh, OK. I thought this is what I always did so I misunderstood your comment regarding this. I'll correct it.

BTW, do you prefer 1-commit pull requests (in which case I overwrite the changes each time and force push, so I guess line comments like this disappear) or many small commits are OK (in which case the "history" of the PR itself is kept but it gets messier when merged).

Comment (hadley) [inline NEWS]: Small commits is fine - if it gets too messy I'll ask you to flatten.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**